pkg/cli/store.go

```diff
@@ -74,6 +74,9 @@ func init() {
 
 	set.StringP("vuln-id", "v", "", "CVE, GHSA or OSV ID to check")
 	set.Int("num-path", 0, "number of paths to return, 0 means all paths")
+	set.String("start-purl", "", "string input of purl with package to start search from")
+	set.String("stop-purl", "", "string input of purl with package to stop search at")
+	set.Bool("sample-data", false, "ingest sample data")
```

Comment:
Can you make `sample-data` more descriptive as we already have another flag for test data ingestipm:

```
	set.Bool("gql-test-data", false, "Populate backend with test data")
```

Reply:
Update:  I am going to remove the ingestion of the test data (including this flag)